rm-datasource/src/main/java/io/seata/rm/datasource/undo/KeywordCheckerFactory.java

```diff
@@ -36,6 +37,10 @@
      * @return keyword checker
      */
     public static KeywordChecker getKeywordChecker(String dbType) {
+        if (JdbcConstants.H2.equalsIgnoreCase(dbType)) {
```

Comment:
How about remove this? And create a new pr to support H2 keyword checker?

Reply:
ok